src/lib/utils/meta_tables/meta_system_information_table.cpp

```diff
@@ -0,0 +1,56 @@
+#ifdef __linux__
+
+#include <numa.h>
+#include <sys/sysinfo.h>
+
+#endif
+
+#include "meta_system_information_table.hpp"
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+MetaSystemInformationTable::MetaSystemInformationTable()
+    : AbstractMetaSystemTable(TableColumnDefinitions{{"cpu_count", DataType::Int, false},
+                                                     {"ram", DataType::Long, false},

```

Comment:
I would call that `system_memory_total_bytes`, also could we add the clock speed?

Reply:
@Bensk1 Remember that this might be different on a per-core basis and might change between calls. Sure you need this?

Reply:
I was talking about the main memory capacity. Why should that change?

Reply:
Isn't the base frequency misleading in the presence of (1) frequency scaling / CPU governors, (2) turbo boost, (3) ZMM (AVX-512) slowdown?

Reply:
I imagined using the base frequency. I haven't seen anybody reporting the ZMM slowdown in her paper yet so it should be fine to leave that out. I could also live with the CPU model name, e.g., `Intel(R) Xeon(R) Platinum 8180 CPU @ 2.50GHz`.

Reply:
I'd be more comfortable with the model name, yes